### PR TITLE
[SUPERMASSIVE] resolvers type error fixed

### PR DESCRIPTION
**Pull request by vejrj**

### CODE REVIEW

File: packages/supermassive/src/__tests__/subscribe.test.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ import models from "../benchmarks/swapi-schema/models";
 import resolvers from "../benchmarks/swapi-schema/resolvers";
 import { addTypesToRequestDocument } from "../ast/addTypesToRequestDocument";
 import { extractImplicitTypes } from "../extractImplicitTypesRuntime";
-import { Resolvers, Resolver } from "../types";
+import { BasicResolvers, Resolvers } from "../types";
```
Comment (freiksenet) [inline packages/supermassive/src/__tests__/subscribe.test.ts:8]: BasicResolvers -> UserResolvers

File: packages/supermassive/src/__tests__/execute.test.ts [hunk 2]
```diff
@@ -301,17 +301,14 @@ async function compareResultsForExecuteWithoutSchema(
...
-  );
   const document = parse(query);
   const result = await executeWithoutSchema({
     document: addTypesToRequestDocument(schema, document),
     contextValue: {
       models,
     },
-    resolvers: fullResolvers,
+    resolvers: resolvers as BasicResolvers,
+    schemaResolvers: (extractedResolvers as any) as Resolvers,
```
Comment (freiksenet) [inline packages/supermassive/src/__tests__/execute.test.ts:311]: Why do we need those type casts?

If we generate typescript, maybe we should annotate generated file to have Resolvers type annotation.

**Review by freiksenet — CHANGES_REQUESTED** [review]

File: packages/supermassive/src/benchmarks/index.ts [hunk 3]
```diff
@@ -66,8 +73,20 @@ queryRunningSuite.add("graphql-jit - precompiled", async () => {
   }
 });
 queryRunningSuite.add("supermassive - runtime schemaless", async () => {
+  let extractedResolvers: Resolvers = {};
```
Comment (vejrj) [inline packages/supermassive/src/benchmarks/index.ts:76]: not sure about this

File: packages/supermassive/src/types.ts [hunk 4]
```diff
@@ -60,17 +60,40 @@ export type InterfaceTypeResolver<
   TContext = any,
   TArgs = any
 > = {
-  [key: string]: FieldResolver<TSource, TContext, TArgs>;
+  __implementedBy: string[];
```
Comment (vejrj) [inline packages/supermassive/src/types.ts:63]: not sure about this

File: packages/supermassive/package.json [hunk 5]
```diff
@@ -14,7 +14,7 @@
   "scripts": {
     "build": "monorepo-scripts build",
     "lint": "monorepo-scripts lint",
-    "test": "supermassive extract-schema ./src/benchmarks/swapi-schema/schema.graphql && monorepo-scripts test",
+    "test": "./bin/supermassive.js extract-schema ./src/benchmarks/swapi-schema/schema.graphql && monorepo-scripts test",
```
Comment (vejrj) [inline packages/supermassive/package.json:17]: and maybe this

**Review by freiksenet — APPROVED** [review]

**Pull request merged**